### PR TITLE
[NO-ISSUE] tooling defect: adjusting the deletion process of an edgeApp and correcting the flow of domains

### PR DESCRIPTION
**Pull request by rafael-qazion**
### What is the new behavior introduced by this PR?
adjusting the deletion process of an edgeApp and correcting the flow of domains

### Does this PR introduce breaking changes?
- [X] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
No

### Does it have a link on Figma?
No

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [X] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [X] Commits are tagged with the right word (feat, test, refactor, etc)
- [X] Application responsiveness was tested to different screen sizes
- [X] Code is formatted and linted
- [X] Tags are added to the PR

#### These changes were tested on the following browsers:
- [X] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: cypress/support/utils.js [hunk 1]
```diff
@@ -1,20 +1,8 @@
...
-      second: '2-digit',
-      millisecond: '3-digit'
-    })
-    .replace(/\//g, '')
-    .replace(/:/g, '')
-    .replace(/, /g, '')
-  return `${prefix}${formattedTimestamp}`
+  const timestamp = new Date();
+  const formattedTimestamp = `${timestamp.getDate().toString().padStart(2, '0')}${(timestamp.getMonth() + 1).toString().padStart(2, '0')}${timestamp.getFullYear()}${timestamp.getHours().toString().padStart(2, '0')}${timestamp.getMinutes().toString().padStart(2, '0')}${timestamp.getSeconds().toString().padStart(2, '0')}${timestamp.getMilliseconds().toString().padStart(3, '0')}`;
+  return `${prefix}${formattedTimestamp}`;
```
Comment (william-tome) [inline cypress/support/utils.js:5]: Acredito que essa mudança não seja necessária. Até porque o método anterior é bem mais legível e o resultado de ambos é o mesmo.

Comment (matheus-sunderhus) [reply cypress/support/utils.js:5]: Pode ser usado o webAPI crypto. que em https gera UUIDs assim como o crypto usado em ambiente server ( Node).
const sufix = window.crypto.randomUUID()

return `${prefix}${sufix}`

**Review by matheus-sunderhus — COMMENTED** [review]
suggestion to use web api crypto.

File: cypress/support/utils.js (hunk 1 above)
Comment (william-tome) [reply cypress/support/utils.js:5]: UUID pode não servir, pois alguns campos na UI/API tem regras em relação a hífens.

**Review by william-tome — APPROVED** [review]

**Review by alisson-marcelo-azion — APPROVED** [review]

**Pull request merged**